assignment2/q1.py

```diff
@@ -0,0 +1,36 @@
+# Print AncestorsGiven a Binary Tree and a key, 
+# write a function that prints all the ancestors of the key in the givenbinary tree.
+class Node:
+ 
+    
+    def __init__(self, data):
+        self.data = data 
+        self.left = None
+        self.right = None
+
+    def printAncestors(root, target):
```

Comment:
Calling the first argument `root` may confuse readers, it's best to always stick with `self`.